src/operators_lazysum.jl

```diff
@@ -153,10 +169,10 @@ end
 identityoperator(::Type{<:LazySum}, ::Type{S}, b1::Basis, b2::Basis) where S<:Number = LazySum(identityoperator(S, b1, b2))
 
 function embed(basis_l::CompositeBasis, basis_r::CompositeBasis, indices, op::LazySum)
-    LazySum(basis_l, basis_r, op.factors, ((embed(basis_l, basis_r, indices, o) for o in op.operators)...,))
+    LazySum(basis_l, basis_r, op.factors, map(o->embed(basis_l, basis_r, indices, o), op.operators))
```

Comment:
Any particular reason you prefer an explicit map to an implicit generator comprehension?

Reply:
Yes - I want it to leave vectors as vectors and tuples as tuples, rather than explicitly constructing a tuple.